generic/tests/cfg/lvm.cfg

```diff
@@ -11,7 +11,8 @@
     image_size_stg2 = 1G
     image_format_stg2 = qcow2
     guest_testdir = /mnt
-    kill_vm = no
+    mount_lv = yes
```

Comment:
@luckyh thanks for your commnets. for sub test "fillup_disk" and "ioquit", may be used in other test case that no need to mount lv, so add "mount_lv=yes/no" in test. I think it is ok.